spec/src/main/asciidoc/architecture.adoc

```diff
@@ -57,18 +57,24 @@ Section <<required-metrics#required-metrics>> lists the required metrics. This l
 These are listed here as they are dependent on the underlying JVM and not the server and thus fit better in _base_ scope
 than the _vendor_ one.
 
+The RESTful metrics are listed as required but do not have to be enabled by default. It is up to the implementation to decide how to enable the REST metrics.
+
 Required base metrics are exposed under `/metrics/base`.
 
 ===== Application metrics
 
 Application specific metrics can not be baked into the server as they are supposed to be provided by the
-application at runtime. Therefore a Java API is provided. Application specific metrics are supposed to be
-portable to other implementations of the MicroProfile. That means that an application written to this
-specification which exposes metrics, can expose the same metrics on a different compliant server
-without change.
+application at runtime. Therefore a Java API is provided.  Application specific metrics are supposed to be portable to other implementations
+of the MicroProfile. That means that an application written to this specification which exposes metrics,
+can expose the same metrics on a different compliant server without change.
 
 Details of this Java API are described in <<app-programming-model#app-programming-model>>.
 
+The Microprofile Metrics specification outlines the implementation of _application_ metrics derived from the usage of REST resources.
+The implementation of these REST metrics are optional.
```

Comment:
This is IMO at odds with the sentence 
> The RESTful metrics are listed as required

I think REST metrics should be optional, in the meaning that if an implementation does not support them at all, it does NOT mean it can't claim to be a MP Metrics implementation